idanalysis/idsdata.py

```diff
@@ -2,8 +2,8 @@
 
 from idanalysis import IDKickMap as _IDKickMap
 
-DATA_REPOS_PATH = "/Dados/"  # Put your data repository path here
-REPOS_PATH = "/home/gabriel/repos/idanalysis/"  # Put your repository path here
+DATA_REPOS_PATH = "/opt/ids-data/"  # Put your data repository path here
+REPOS_PATH = "~/repos/idanalysis/"  # Put your repository path here
```

Comment:
since this package is a library, it is not good practice to have paths hardcoded this way. It's better to define a interface for the user to set the paths themselves.